src/client.ts

```diff
@@ -227,6 +227,7 @@ import {
     CryptoApi,
     decodeRecoveryKey,
     ImportRoomKeysOpts,
+    CryptoEvent as RustCryptoEvent,
```

Comment:
:-1: to this name. `CryptoApiCryptoEvent`, possibly. Or, better, rename the old one to `LegacyCryptoEvent`.